docs/usage/appending-overwriting-delta-lake-table.md

```diff
@@ -0,0 +1,76 @@
+# Appending to and overwriting a Delta Lake table
+
+This section explains how to append to an exising Delta table and how to overwrite a Delta table.
+
+## Delta Lake append transactions
+
+Suppose you have a Delta table with the following contents:
+
+```
++-------+----------+
+|   num | letter   |
+|-------+----------|
+|     1 | a        |
+|     2 | b        |
+|     3 | c        |
++-------+----------+
+```
+
+Append two additional rows of data to the table:
+
+```python
+df = pd.DataFrame({"num": [8, 9], "letter": ["dd", "ee"]})
+dl.writer.write_deltalake("tmp/some-table", df, mode="append")
```

Comment:
Should we showcase examples where you import it like as dl. 

I think we generally import directly, right?